update proteinpaint-client to 2.169.0

siosonel opened the pull request:
## Description

@craigrbarnes I had to add an override rule in `.eslintrc` to make the auto-generated `next-env.d.ts` work with triple-slash, instead of getting a linting error.

Fixes
- [x] [SV-2627](https://gdc-ctds.atlassian.net/browse/SV-2627) -fixed by https://github.com/stjude/proteinpaint/pull/4045
- [x] [SV-2679](https://gdc-ctds.atlassian.net/browse/SV-2679) - fixed by https://github.com/stjude/proteinpaint/pull/4032
- [x] [SV-2696](https://gdc-ctds.atlassian.net/issues?filter=13139&selectedIssue=SV-2696) - fixed by https://github.com/stjude/proteinpaint/pull/4035


## Checklist

- [ ] Added proper unit tests
- [ ] Left proper TODO messages for any remaining tasks
- [ ] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

## Screenshots/Screen Recordings (if Appropriate)

siosonel committed:
update proteinpaint-client to 2.169.0

craigrbarnes reviewed (changes requested):
Can you remove/rollback the changes to next-env.d.ts, as we are going to remove it from being checked in? It's already in .gitingore as part of a recent PR.

siosonel committed:
revert changes to eslintrc.js and portal-proto/next-env.d.ts

siosonel commented:
@craigrbarnes (EDIT: I'll go ahead and add back the override entry to `.eslintrc`) Should I keep the `override` entry that I added to `.eslintrc.js`?  I've also removed it in my previous commit, but I think that may be causing the gitlab check failure (I'm not able to see the logs).

siosonel committed:
add back the override entry to `.eslintrc.js`

craigrbarnes commented:
> @craigrbarnes (EDIT: I'll go ahead and add back the override entry to `.eslintrc`) Should I keep the `override` entry that I added to `.eslintrc.js`? I've also removed it in my previous commit, but I think that may be causing the gitlab check failure (I'm not able to see the logs).

The override entry will be fine.  Also, we have some additional work to do on removing  next-env.d.ts, so you can go ahead and add it back so we can get the pipeline to pass. Sorry for the confusion

siosonel committed:
add back auto-generated comments in next-env.d.ts

siosonel commented:
@wteouchicago Not sure if I still need to request manual deployment of this release to qa-int, but any help would be appreaciated, thanks as always!

wteouchicago commented:
> @wteouchicago Not sure if I still need to request manual deployment of this release to qa-int, but any help would be appreaciated, thanks as always!

@siosonel no problem! I'll keep an eye out over the next 1-2 hours and if it's not automatically deployed, I'll ask for a manual deployment.

wteouchicago commented:
@siosonel I think I see the PP changes in qa-int now. FYI.
